mainwindow.cpp

```diff
@@ -52,8 +52,13 @@ MainWindow::MainWindow(std::shared_ptr<flasher::Flasher> flasher, QWidget *paren
     DisableAllButtons();
     ClearProgress();
 
-    connect(flasher_.get(), &flasher::Flasher::UpdateProgress, this, [&] (const qint64& sent_size, const qint64& firmware_size) {
-        const int progress_percentage = (100 * sent_size) / firmware_size;
+    connect(flasher_.get(), &flasher::Flasher::UpdateProgressBar, this, [&] (const qint64& sent_size, const qint64& firmware_size) {
+
+        int progress_percentage = 0;
+        if(firmware_size != 0) {

```

Comment:
Space after if is needed.

Reply:
again... I'll need to find time to see how to connect Astyle to the githook.